IDEASAcquaman_internal.pro

```diff
@@ -13,8 +13,6 @@ HEADERS +=	source/application/IDEAS/IDEASAppController.h \
 	source/acquaman/IDEAS/IDEASXASScanConfiguration.h \
 	source/ui/IDEAS/IDEASXASScanConfigurationView.h \
 	source/beamline/IDEAS/IDEASKETEKDetector.h \
-	source/acquaman/IDEAS/IDEASXRFScanConfiguration.h \
-	source/acquaman/IDEAS/IDEASXRFScanController.h \
 	source/ui/IDEAS/IDEASXRFDetailedDetectorView.h \
```

Comment:
It looks as if a lot of the other beamline specific XRF detailed detector views have been removed, but this one remains. Is there something specific to this one that means it has to stay?